### PR TITLE
fix: max-width and reduce padding

### LINKED ISSUE #660: Visual tweaks for CH3 Challenge 4
From @GBKS in #653 

Thanks for fixing the skip, it's gone now.

I noticed a couple more visual things to tweak. Totally cool to do those in a separate PR.

image

There are still some fixed widths in the code. Those will likely cause layout issues when text size changes with localizations.

### PR DESCRIPTION
**Pull request by 0tuedon**
This PR Closes #660 

Added a max-width only on challenge 4 so it does not affect other profile containers in other chapters

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
@GBKS Would you mind taking a look at this PR and reviewing it?

**Comment by GBKS** [comment]
I am running into a weird issue. When I run this branch locally, the left profile containers are cut off, even when I am scrolled all the way to the left. That is due to a "justify-content: center" attribute on the container of those profiles. But this issue does not show up on the GitHub/Vercel preview link. How could that be?

image

**Comment by 0tuedon** [comment]
> I am running into a weird issue. When I run this branch locally, the left profile containers are cut off, even when I am scrolled all the way to the left. That is due to a "justify-content: center" attribute on the container of those profiles. But this issue does not show up on the [GitHub/Vercel preview link](https://saving-satoshi-git-fork-0tuedon-visual-twe-a195a5-savingsatoshi.vercel.app/en/chapters/chapter-3/split-2?dev=true). How could that be?
> 
> image

This is weird,  i don't currently get this issue on local, have you tried refreshing on local or restarting the server to try again.

**Comment by GBKS** [comment]
The issue I have is not related to your code changes and this PR, since I also see it on the dev branch. Let's tackle it separately.

It looks like this code in `ProfilesContainer.tsx` is never getting called.
image

File: ui/common/ProfilesContainer.tsx [hunk 1]
```diff
@@ -39,14 +39,15 @@ export default function ProfilesContainer({
       }
     }
   }, [])
-
+  console.log(className, 'class Name test')
```
Comment (GBKS) [inline ui/common/ProfilesContainer.tsx:42]: Please remove this log.

File: ui/common/ProfilesContainer.tsx [hunk 2]
```diff
@@ -39,14 +39,15 @@ export default function ProfilesContainer({
...
   return (
     <>
-      <div className="flex flex-col items-center gap-[30px] self-stretch">
+      <div className=" flex  flex-col items-center gap-[30px] self-stretch">
```
Comment (GBKS) [inline ui/common/ProfilesContainer.tsx:45]: No need for those extra spaces.

**Comment by 0tuedon** [comment]
> The issue I have is not related to your code changes and this PR, since I also see it on the dev branch. Let's tackle it separately.
> 
> It looks like this code in `ProfilesContainer.tsx` is never getting called. image

Okay

**Review by GBKS — APPROVED** [review]
Looks great. Thank you.

**Review by benalleng — APPROVED** [review]
code LGTM!

**Pull request merged**